cli/src/semgrep/app/scans.py

```diff
@@ -1,7 +1,9 @@
 # Handle communication of findings / errors to semgrep.app
 import json
+import logging
```

Comment:
```suggestion
from semgrep.verbose_logging import getLogger

logger = getLogger(__name__)
```
is recommended for cli use. It adds a log level for `--verbose`

Reply:
I'm importing `logging` here for logging.DEBUG since it's not in `verbose_logging`
`getLogger` is imported and used as you suggested after the imports.

Reply:
Cleaned up the import to be more specific